v2/fleet/mesos-slave.service

```diff
@@ -33,7 +31,6 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
     --containerizers=docker,mesos \
     --executor_registration_timeout=10mins \
     --hostname=`curl -s http://169.254.169.254/latest/meta-data/public-hostname` \
-    --isolation=cgroups/cpu,cgroups/mem \
```

Comment:
not really. we don't have numbers either way. Switching back to the default *seems* like the safer thing to do.